src/main/java/jsr223/perl/utils/Log4jConfigurationLoader.java

```diff
@@ -25,7 +25,7 @@
  */
 package jsr223.perl.utils;
 
-public class PerlLog4jConfigurationLoader {
+public class Log4jConfigurationLoader {
```

Comment:
In docker-compose, this class doesn't have the prefix "DockerCompose". Moreover, if there is no error in the scheduler package name should be enough to differentiate the class.